GridKit/Model/PhasorDynamics/Branch/Branch.hpp

```diff
@@ -165,6 +173,9 @@ namespace GridKit
       /* Derivied parameters */
```

Comment:
```suggestion
      /* Derived parameters */
```
not the result of this pr, but since it's right here

Reply:
done